Makefile

```diff
@@ -3,29 +3,29 @@ TRAVIS_BUILD_NUMBER?=debug0
 
 .PHONY: coverage get test clean
 
-all: get coverage AWSnycast
+all: _vendor coverage AWSnycast
 
-AWSnycast: *.go */*.go
-	go build -a -tags netgo -ldflags '-w' .
-	strip AWSnycast
+_vendor: Gomfile
+	gom install
 
-test:
-	go test -short ./...
+AWSnycast: *.go */*.go _vendor
+	gom build -a -tags netgo -ldflags '-w' .
+	strip AWSnycast
```

Comment:
ARGH, this fails on Darwin, but it makes things significantly smaller on x86, so I don't really want to remove it.

Maybe I should just move it to the packaging step (as we only run fpm etc on Linux)